juniper_official/Solutions/EX-Platform/DHCP-server-statistics.playbook

```diff
@@ -0,0 +1,17 @@
+/*
+ * Playbook contains 2 rules which monitors dhcp local server and dhcp relay statistics
+ * and notifies when anomalies are found.
+ * 
+ * 1) Rule dhcp.localserver/check-statistics-iagent, detects the dhcp counters and notifies
+ *    anomalies when count exceeds threshold values.
+ * 2) Rule dhcp.relayserver/check-statistics-iagent, detects the dhcp decline value and notifies
+ *     anomalies when count exceeds threshold value.
+ *
```

Comment:
1. have more meaningful name for both the rules e.g.. check-dhcp-server-statistics-netconf
2. replace iagent with netconf in all rule names.

Reply:
updated